app/views/return-requirements/requirements-approved.njk

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set rootLink = "/licences/" + licenceId %}
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'back',
+      href: rootLink + "/no-return-check-your-answers"
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+   {{ govukPanel({
+      titleText: "Returns requirements approved",
+      classes: "govuk-!-margin-bottom-4"
+    }) }}
+
+    <h2 class="govuk-heading-m">What happens next</h2>
+    <p class="govuk-body">These return requirements are now active and returns will be opened for this licence in the next returns cycle.</p>
+    <p class="govuk-body"><a href="{{rootLink}}" class="govuk-link">Return to licence set up</a></p>
+{% endblock %}
```

Comment:
If only to avoid this scary-looking glyph, we ensure all files end with a newline.

![Screenshot 2023-12-12 at 16 41 24](https://github.com/DEFRA/water-abstraction-system/assets/1789650/a8ba05c7-e8a7-444d-9de0-26ffadf707e7)

If it helps there is a setting you can enable in VSCode to ensure this always happens when you save a file, `"files.insertFinalNewline": true,`. We have a few suggested VSCode settings folks might want to enable. Check out any of the `*.settings.json` files in `docs/vscode` to see them. You can also run `🌈 Settings` in **wal-dev-environment** to have these settings applied to all the repos.

```suggestion
{% endblock %}

```

Reply:
Ok, I normally have this set to true or get eslint to catch in pre-commit hooks. 

Added in new line :)